packages/sources/finage/src/adapter.ts

```diff
@@ -1,41 +1,36 @@
-import { Execute } from '@chainlink/types'
+import { ExecuteWithConfig, ExecuteFactory, Config} from '@chainlink/types'
 import { Requester, Validator, AdapterError } from '@chainlink/ea-bootstrap'
 import { util } from '@chainlink/ea-bootstrap'
-
-export const NAME = 'Finage'
+import { makeConfig, NAME } from './config'
 
 const customParams = {
   base: ['base', 'from', 'symbol'],
   endpoint: false,
 }
 
-const baseUrl = 'https://api.finage.co.uk'
-
-const DEFAULT_ENDPOINT = 'stock'
```

Comment:
We lost this on resolving the merge conflict. Let's bring it back.

And line 21 where it plugs in:
```
  const endpoint = validator.validated.data.endpoint || DEFAULT_ENDPOINT
```

Reply:
I'm not sure why we need the `baseURL` part because isn't that spread in through `...config.api```